leda/src/SuggestionList/NoSuggestions.tsx

```diff
@@ -2,6 +2,6 @@ import * as React from 'react';
 import { Div } from '../../components/Div';
 import { NoSuggestionsProps } from './types';
 
-export const NoSuggestions = ({ className }: NoSuggestionsProps): React.ReactElement => (
-  <Div className={className}>Ничего не найдено</Div>
+export const NoSuggestions = ({ className, noSuggestionsText = null }: NoSuggestionsProps): React.ReactElement => (
+  <Div className={className}>{ noSuggestionsText }</Div>
```

Comment:
Are you sure we have to render empty div here in that case?

Reply:
Yes, it adds a wrapper with css class nodata which positions the content in the center and colors the text
But maybe in this case some default text would be better than no text
And if there is no text, yes. it looks unnecessary
So bringing the text back?
E.g. Nothing found

Reply:
Sounds good to me, let's do default!)